algopy_testing/src/algopy_testing/gtxn.py

```diff
@@ -0,0 +1,6 @@
+class AssetTransferTransaction:
```

Comment:
At the moment this allows the mock to use invalid types, should make this have the appropriate fields so mocks will be type checked correctly. for now just add the fields required for the examples test with a TODO for the others